submissions/ORybka/post-a-tiny-js-world/index.js

```diff
@@ -0,0 +1,61 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/ORybka/a-tiny-JS-world
+   Web app: https://orybka.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
```

Comment:
No all inhabitants have hands by natural design. Not that they just have `0` hands or `null` hands or `undefined` number of hands, but they do not hands property at all.
Imagine, we add birds. Under current paradigm every inhabitant would have wings, just some of them would have zero wings. This way the `Inhabitant` class becomes too general, and not easy to maintain.

The beauty of OOP is that every base class owns and handles only shared props, and every child class introduces and handles props unique to that child class.

Handling of props:
a child class can have its own method (e.g. `methodName`) where it calls parent's method (e.g. `super.methodName()`) and supplements it with some data from unique properties it **owns** directly.

Reply:
@OleksiyRudenko  Thank you for this advice!
I am not sure whether my solution is good, but I can't solve it better now :( 
It is very helpful that you can make your own methods for children based on the parent's methods but for now I can't understand how to add `hands` to `super.getValues()`

Reply:
@ORybka you do not need to pass anything to a super method if it doesn't require.
```
class A {
  constructor() {
  }
  method() {
    return 'a'; // "a"
  }
}

class B extends A {
  constructor() {
    super();
  }
}


class C extends B {
  constructor() {
    super();
  }
  method() {
    return super.method() + 'c'; // "ac"
  }
}
```

Consider `super` a substitute to the name of a class nearest to current class that has `method` defined.
It's a shortcut that allows you e.g. insert interim classes without rewriting (which would be required if we used notation like `A.super()`)

`super()` is a shortcut for `super.constructor()`